katib/katib-controller/overlays/ibm-storage-config/katib-db-deployment.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: katib-db
+spec:
+  template:
+    spec:
+      containers:
+      - name: katib-db
+        livenessProbe: null
```

Comment:
A comment explaining why livenessProbe is disabled would be helpful.

Reply:
I added a comment in my latest commit, we are getting some segmentation fault when the livenessProbe hit our db pod.